README.md

```diff
@@ -6,6 +6,8 @@
 # elm-in-elm
 Elm compiler written in Elm!
 
+<a href="https://travis-ci.com/elm-in-elm/compiler" alt="Build Status">
+  <img src="https://travis-ci.com/elm-in-elm/compiler.svg?branch=master" /></a>
```

Comment:
Oh, also one more thing: let's use the shields.io badge:
```
https://img.shields.io/travis/com/elm-in-elm/compiler/master.svg
```

The original one has a bit different style (at least for me in Firefox@Linux) so if we can, let's make it consistent :)